mlflow/projects/backend/local.py

```diff
@@ -180,8 +193,11 @@ def _invoke_mlflow_run_subprocess(
         run_id=run_id,
         parameters=parameters,
     )
-    env_vars = get_run_env_vars(run_id, experiment_id)
-    env_vars.update(get_databricks_env_vars(mlflow.get_tracking_uri()))
+
+    env_vars = get_run_env_vars(run_id, experiment_id, tracking_uri)
+    env_vars.update(
+        get_databricks_env_vars(tracking_uri=tracking._resolve_tracking_uri(tracking_uri))
+    )
```

Comment:
Suggestion to replace with with
```python
def get_databricks_run_env_vars(run_id, experiment_id, tracking_uri):
  env_vars = get_run_env_vars(run_id, experiment_id, tracking_uri)
  env_vars.update(get_databricks_env_vars(tracking_uri=tracking._resolve_tracking_uri(tracking_uri)))
  return env_vars
```

Reply:
Creating a helper function sounds great!